lib/src/extensions.dart

```diff
@@ -30,6 +30,37 @@ extension DateTimeTimeExtension on DateTime {
 
   /// Subtracts the Duration from this DateTime returns the difference as a new DateTime object.
   DateTime operator -(Duration duration) => subtract(duration);
+
+  /// Returns a range of dates to [to], exclusive start, inclusive end
+  /// ```dart
+  /// final start = DateTime(2019);
+  /// final end = DateTime(2020);
+  /// start.to(end, by: const Duration(days: 365)).forEach(print); // 2020-01-01 00:00:00.000
+  /// ```
+  Iterable<DateTime> to(DateTime to,
```

Comment:
I think this might break on the CI. Line width should be set to 120.

Reply:
Funny, I use 120 as well, but just assumed everyone else uses 80.

Reply:
Done

Reply:
❤️